### PR TITLE
WIP: Added CDF for gamma, inverse gamma and log normal densities

### PR DESCRIPTION
**Pull request by omarfsosa**
Just a couple of nice-to-have methods.

### CODE REVIEW

File: numpyro/distributions/continuous.py
Comment (fehiepsi) [inline numpyro/distributions/continuous.py:456]: nit: I guess you can use `cdf(1 / x)` for simplicity (and a bit faster)

File: numpyro/distributions/continuous.py
Comment (fehiepsi) [inline numpyro/distributions/continuous.py:858]: nit: maybe using `cdf(log(x))` for consistency with the above one

**Review by fehiepsi — COMMENTED** [review]
Thanks, @omarfsosa! It is great to have those cdf methods!

**Comment by fehiepsi** [comment]
Hi @omarfsosa, you might want to merge the fixes in #1251 to unblock your PR.

**Comment by omarfsosa** [comment]
> 

Ah thanks, I was wondering what happened there. I'll merge those.
Thanks for the suggestions too, I'll fix that and update my PR.

**Comment by fehiepsi** [comment]
@omarfsosa There are a couple of failing tests due to numerical issues and a weird failing test for vectorized chain method. But I have addressed them in that PR and it has been merged to master. You might want to merge with `master` branch to get the tests to pass.

**Comment by fehiepsi** [comment]
Hi @omarfsosa, could you merge the master branch? CI is not stable recently. We just added a couple of other fixes for CI. I think after merging, this PR should pass CI.

**Review by fehiepsi — APPROVED** [review]
Thanks, @omarfsosa !

**Pull request merged**
